typescript-objects/main.ts

```diff
@@ -0,0 +1,62 @@
+interface StudentProps {
+  firstName: string;
+  lastName: string;
+  age: number;
+  livesInIrvine?: boolean;
+  previousOccupation?: string;
+}
+
+interface Vehicle {
+  make: string;
+  model: string;
+  year: number;
+  color?: string;
+  isConvertible?: boolean;
+}
+
+interface Pet {
+  name?: string;
+  kind?: string;
+}
+
+const student: StudentProps = {
+  firstName: 'Katelyn',
+  lastName: 'Ngo',
+  age: 28,
+  livesInIrvine: false,
+  previousOccupation: 'teacher',
```

Comment:
You should remove these two lines because you are adding these values on lines 36 and 37